source/documentation/deploying_services/s3.md

```diff
@@ -179,21 +179,25 @@ You can take different actions on an S3 bucket using the [AWS S3 API](https://do
 
 ### Access objects in a public AWS S3 bucket
 
-Once you have uploaded an object to a [public S3 bucket](#provision-a-public-aws-s3-bucket), you can find that object at: 
+Once you have uploaded an object to a [public S3 bucket](#provision-a-public-aws-s3-bucket), you can find that object at:
 
 ```
 https://BUCKET_NAME.s3.amazonaws.com/KEY
 ```
 
 where:
 
-- `BUCKET_NAME` is the `bucket_name` value in the bucket [environment variables](/deploying_apps.html#system-provided-environment-variables) 
+- `BUCKET_NAME` is the `bucket_name` value in the bucket [environment variables](/deploying_apps.html#system-provided-environment-variables)
 - `KEY` is the key you gave to the object when uploading that object to the bucket
 
 ### Access objects in a private AWS S3 bucket using presigned URLs
 
-You can [generate a presigned URL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html)
-that gives temporary access to an object in a private bucket.
+You can [generate a presigned URL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html) that gives temporary access to an object in a private bucket. Presigned URLs are useful in scenarios where you want to:
+
+* [grant limited access to a resource in the bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html) after validating the user's authorization (for example, downloading a signed document)
```

Comment:
> after validating the user's authorization
Is this the user who creates the URL or the user who has temporary access? I assume the former, but I think it's not clear

Also:
```suggestion
* [grant limited access to a resource in the bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html) after validating the user's authorisation (for example, downloading a signed document)
```

Reply:
> Is this the user who creates the URL or the user who has temporary access? I assume the former, but I think it's not clear

I meant the user of the application, and the application applying its domain logic to see if that user should be allowed to access the thing.